src/System.Private.ServiceModel/src/System/IdentityModel/Claims/X509CertificateClaimSet.cs

```diff
@@ -339,6 +375,59 @@ public IIdentity Identity
                 get { return _identity; }
             }
         }
+
+        // We don't have a strongly typed extension to parse Subject Alt Names, so we have to do a workaround 
+        // to figure out what the identifier, delimiter, and separator is by using a well-known extension
+        private static class X509SubjectAlternativeNameConstants
+        {
+            public const string Oid = "2.5.29.17"; 
+            public static readonly string Identifier;
+            public static readonly char Delimiter;
+            public static readonly string Separator;
+
+            // static initializer runs only when one of the properties is accessed
```

Comment:
@iamjasonp Yep, `const` values are embedded as literals at the callsite, the type isn't involved at runtime.